### DISCUSSION TITLE
Tap action to open the calendar in a dialog/modal

### DISCUSSION THREAD

**Discussion by sroettering** [totaldebug/atomic-calendar-revive#1300] [Feature Requests]
I'm using the calendar in event mode on my wall tablet dashboard displaying only events for "today". Now when I'm standing in front of the tablet and want to check what's up this week for example I would love to have an option to open the calendar in the calendar mode (hope that's what it's called) to get an easy overview. My proposal would be to add a more-info tap action that opens in a dialog box showing the whole calendar.

**Comment by marksie1988** [comment]
duplicate of https://github.com/totaldebug/atomic-calendar-revive/discussions/1389
